crates/settings/src/general.rs

```diff
@@ -0,0 +1,30 @@
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Clone, Default, Debug)]
+pub struct GeneralSettings {
+    pub api_token: Option<String>,
+    pub sentry: Sentry,
+}
+
+impl GeneralSettings {
+    pub fn is_sentry_enabled(&self) -> bool {
+        self.sentry.0 .0
+    }
+
+    pub fn sentry_dsn(&self) -> String {
+        self.sentry.0 .1.clone()
+    }
+}
+
+// enabled, dsn
+#[derive(Serialize, Deserialize, Clone, Debug)]
+pub struct Sentry(pub (bool, String));
```

Comment:
I'd use explicit fields for this to make it obvious what they represent, especially for people editing the config